README.md

```diff
@@ -36,37 +36,40 @@ The software is finished -- and by finished, we mean there are no show-stopping,
 
 Here is a more detailed [roadmap](https://github.com/OrchardCMS/OrchardCore/wiki/Roadmap).
 
-## Getting Started
+## Getting Started and Documentation
 
-- Clone the repository using the command `git clone https://github.com/OrchardCMS/OrchardCore.git` and checkout the `main` branch.
+The documentation can be accessed here: <https://docs.orchardcore.net/>. See [the getting started docs](https://docs.orchardcore.net/en/latest/docs/getting-started/) on how to start using Orchard Core.
 
-### Command Line
+You can also run Orchard Core from Docker:
```

Comment:
It would be nice to list the required steps to install then run the docker image

Reply:
That's explained under the linked https://docs.orchardcore.net/en/latest/docs/topics/docker/ but I added a pointer.